db/migrate/20210118162031_add_error_to_data_update_script.rb

```diff
@@ -0,0 +1,5 @@
+class AddErrorToDataUpdateScript < ActiveRecord::Migration[6.0]
+  def change
+    add_column :data_update_scripts, :error, :string
```

Comment:
Added an error column that will save the error from a failing data update script

Reply:
Do we want to make this a text field in case we want to put more in it?